### PR TITLE
add higress to projects

### PR DESCRIPTION
**Pull request by johnlanni**

### CODE REVIEW

File: site/data/projects.yaml [hunk 1]
```diff
@@ -57,6 +57,14 @@
     Gloo Edge is an open-source function gateway built on top of the Envoy Proxy. Gloo Edge provides a unified entry point
     for access to all services and serverless functions, translating from any interface spoken by a client to any interface
     spoken by a backend.
+- name: Higress
+  logo: higress.svg
+  website: https://higress.io/en-us/
+  github: alibaba/higress
+  description: |
+    Higress is an open-source cloud-native gateway built on top of the Envoy Proxy and Istio. Higress can discovery from
```
Comment (phlax) [inline site/data/projects.yaml:65]:
```suggestion
    Higress is an open-source cloud-native gateway built on top of the Envoy Proxy and Istio. Higress can perform discovery from
```

Comment (phlax) [reply site/data/projects.yaml:65]: also ...

>  built on top of the Envoy Proxy and Istio

doesnt read quite correct, lets lose the "the"

**Comment by johnlanni** [comment]
@phlax Thanks for your adivce, it has been modified.

File: site/data/projects.yaml [hunk 2]
```diff
@@ -57,6 +57,14 @@
...
     for access to all services and serverless functions, translating from any interface spoken by a client to any interface
     spoken by a backend.
+- name: Higress
+  logo: higress.svg
+  website: https://higress.io/en-us/
+  github: alibaba/higress
+  description: |
+    Higress is an open-source cloud-native gateway built on top of Envoy Proxy and Istio. Higress can perform discovery from
+    various service registries, such as Nacos, ZooKeeper, Consul, Eureka, etc. Higress can also function as a feature-rich
+    Kubernetes Ingress controller, which is compatible with many annotations of nginx ingress controller.
```
Comment (phlax) [inline site/data/projects.yaml:67]:
```suggestion
    Kubernetes ingress controller, which is compatible with many annotations of Nginx ingress controller.
```

**Review by phlax — COMMENTED** [review]
on more tiny nit, and this should be good to land

**Review by phlax — APPROVED** [review]
lgtm, thanks @johnlanni

**Pull request merged**